.github/workflows/_reusable_pr-antora-content-guidelines-checker.yml

```diff
@@ -0,0 +1,18 @@
+# Permissions required by this workflow that MUST be set in the calling workflow as this workflow can only downgrade permissions (https://docs.github.com/en/actions/using-workflows/reusing-workflows#supported-keywords-for-jobs-that-call-a-reusable-workflow)
+# pull-requests: write / "pr-antora-content-guidelines-checker" write PR comments when the PR doesn't match the "Guidelines"
+name: Check Antora content guidelines in Pull Request
+
+on:
+  workflow_call: # this allows the workflow to be reused
+
+jobs:
+  check_contribution:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check
+        uses: bonitasoft/actions/packages/pr-antora-content-guidelines-checker@v2
+        with:
+          attributes-to-check: ':description:'
```

Comment:
@benjaminParisel In the initial commit of https://github.com/bonitasoft/bonita-labs-doc/pull/149/commits/7e708e4c604800bf79f8145090c7698ef0005871#diff-7471ecfdb72d7fd63aba8e4156b497c7af8b7a935c3fbc4fd82d362ddd46ab15R14, you added `:page-aliases:`. It seems to me that this would require that all pages require to have this attribute while only those which moved or were deleted require it. This is not in place in the configuration that is currently used in the bonita-doc repository: https://github.com/bonitasoft/bonita-doc/blob/2229384423f83153bd73135f5832c769072a8639/.github/workflows/contribution-checks.yml#L16

I had in mind that this is something that the "pr-antora-content-guidelines-checker" will do in the future by implementing a specific step (that could eventually be disabled).

Are you OK with this analysis or did I missed something?

Reply:
Yeah i'm agree, for now, we just needed to check if :description: is in place. For page-aliases attribute, we need to add a another step to check it only when we rename/delete a page